crates/sui-deepbook-indexer/config.yaml

```diff
@@ -0,0 +1,22 @@
+# config.yaml format:
+# Database connection URL
+# db_url: postgres://dariorussi:dariodb@localhost:5432/deepbook
+db_url: postgres://<USER>:<DB>@localhost:5432/<TABLE>
+
+# URL of the remote store
+remote_store_url: http://checkpoints.testnet.sui.io
+# remote_store_url: http://sjc-bnt-rpc-00.mystenlabs.com:9000/rest
+# Path to the checkpoints
+# checkpoints_path: /Users/dariorussi/tmp/deepbook_indexer/checkpoints
+checkpoints_path: <local_dir>/checkpoints
+# Number of concurrent operations
+concurrency: 1
+
+# Client metric URL
+metric_url: "0.0.0.0"
+# Client metric port
+metric_port: 1000
+
+# Optional starting and ending checkpoints
+start_checkpoint: 88912150
+end_checkpoint: 89800000
```

Comment:
this checkpoint range should have deepbook transactions somewhere. You can adjust it to whatever you want and get transaction for testing very quickly.
For instance you can submit a chunk of transactions, track the checkpoints and you have a very short span of checkpoints with enough transactions to develop